components/admins/modals/descriptionModal.tsx

```
@@ -1,33 +1,56 @@
 
-export function DescriptionModal({ modalId, details, type }: Readonly<DescriptionModalProps>) {
+export function DescriptionModal({
+  open,
+  onClose,
+  details,
+  type,
+}: Readonly<DescriptionModalProps>) {
+  if (!open) return null;
+
+  function handleClose() {
+    onClose && onClose();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)